deposit/hal/metadata.py

```diff
@@ -148,7 +148,10 @@ def render(self, paper, filename, form):
         language.attrib['ident'] = 'en'  # TODO adapt this?
         textClass = addChild(profileDesc, 'textClass')
 
-        domains = [form.cleaned_data['topic'].lower()]
+        domains = []
+        if 'cleaned_data' in dir(form):
```

Comment:
`cleaned_data` is always in `form` in theory.
The problem is that: `topic` might not be in the `cleaned_data` or that `form.is_valid` has not been called to create the `cleaned_data` object.

I don't think it'd be a good idea to check for it, this way.